### ISSUE TITLE
Checkbox not showing / function doesn't exist

### ISSUE THREAD

**Issue by Bobz-zg** [stcr/subscribe-to-comments-reloaded#247]
Hi,

I have an issue with the plugin, at first I thought it was caching problem but turns out not after all.
Plugin is active and I set in Options not to show form, then I manually add it to my comments template:

```
<?php if (function_exists('subscribe_reloaded_show')) :
        echo 'exists';
        subscribe_reloaded_show();
else :
        echo 'doesnt exist';
endif; ?>
```

Sadly this shows that function doesn't exits at all.

If I remove the function from template, and then try to activate the plugin trough options then I can see only part saying: 'You can manage the subscriptions of this post.' but not the checkbox.

![screen shot 2016-08-25 at 20 10 05](https://cloud.githubusercontent.com/assets/420604/17980555/1df11a88-6b00-11e6-9138-1ec2ec4b2664.png)
![screen shot 2016-08-25 at 20 10 46](https://cloud.githubusercontent.com/assets/420604/17980554/1df01a34-6b00-11e6-9518-f1c12d040850.png)

Versions:
- WP 4.6
- PHP 7
- Plugin Version 160115

**Comment by Reedyseth** [comment]
Hello @Bobz-zg ,

Yes there is a bug with that version that I already fixed. This weekend I will be releasing the new update, but it will be helpful if you could test it before.

You can download it from here:

https://github.com/stcr/subscribe-to-comments-reloaded/archive/development.zip

**Comment by Bobz-zg** [comment]
Hy @Reedyseth 
Thanks for quick reply, I will give it a test and let you know how it went.

**Comment by Reedyseth** [comment]
> Thanks for quick reply, I will give it a test and let you know how it went.

Thank you. I will appreciate it !

**Comment by Bobz-zg** [comment]
Hi @Reedyseth 

I have checked and now it's working for me.
Have found one problem, when I want to get to subscription page manager_link shortcode is not parsed, not sure is it me or not. I do get it in email but it's not shown on page after form submit

<img width="647" alt="screen shot 2016-08-26 at 09 44 34" src="https://cloud.githubusercontent.com/assets/420604/17998484/ff0f5d3a-6b74-11e6-93c2-e69b45e761b5.png">

Second thing is a request if possible, and that's to update markup on management page so I can style them to my need, right now it's difficult to style it and it looks bad for me:

![screen-shot-2016-08-26-at-09 53 21]
(https://cloud.githubusercontent.com/assets/420604/17998579/76788806-6b75-11e6-8d4b-a7976b690370.jpg)

1. Remove nbsp
2. Proposed markup:

```
<p class="stcr-p stcr-email">
	<span class="title">Email<span> <span class="email">me@gmail.com</span>
</p>


<div class="stcr-legend">
	<p class="stcr-p stcr-title">Title</p>
	
	<dl>
		<dt>Y<dt>
		<dd>all comments</dd>
		<dt>R</dt>
		<dd>replies only</dd>
		<dt>C<dt>
		<dd>inactive</dd>
	<dl>
</div>

<div class="stcr-actions">
	<p class="stcr-p stcr-title">Title</p>

	<p class="stcr-p stcr-action-name">
		<label for="action-name">
			<span class="title">title</span>
			<input type="radio" name="action-name" />
		</label>
	</p>
</div>
```

**Comment by Reedyseth** [comment]
@Bobz-zg Thank you for the suggestion on the markup. Yes that view needs a lot of UI improvement, as well as responsive support.

> Have found one problem, when I want to get to subscription page manager_link shortcode is not parsed, not sure is it me or not. I do get it in email but it's not shown on page after form submit

Yes that part is buggy because it used to display the manager link, but you could hack this very easy by entering any user email and get the manager link, this is why I removed the functionality for That Tag. 

The solution is to add two sections, 1 for the frontend message and another one for the email notification. I might delay the release of the update a few days to get this kind of bugs fixed.

Thank you for your suggestions.

**Comment by Reedyseth** [comment]
@Bobz-zg 

I have added the solution as I mentioned, please give another try to the development code by downloading againg the files.

https://github.com/stcr/subscribe-to-comments-reloaded/archive/development.zip

**Comment by Bobz-zg** [comment]
@Reedyseth 

Have tested and still have the same issue with manage_link shortcode not parsing.
Looks like you need to: `<?php echo do_shortcode($content); ?>`


Also, another issue is now present, I got empty email that should send me link to manage my comments

**Comment by Reedyseth** [comment]
Did  you noticed that now there is an extra field on the notification options?

One Field is to display the notification request to a user and it only parse the `[blog_name]` tag.

![image](https://cloud.githubusercontent.com/assets/1228711/18028344/79aebeae-6c39-11e6-8e48-3a4cea1ae026.png)

The other field is use for the email notification, in that field you can use the `[blog_name]` and `[manager_link]`

![image](https://cloud.githubusercontent.com/assets/1228711/18028347/91e800d4-6c39-11e6-9260-a15d9623a586.png)

**Comment by Reedyseth** [comment]
@Bobz-zg  A new version is out so please update and check it out !

**Comment by Bobz-zg** [comment]
@Reedyseth I will give it a spin. Thanks.

**Comment by Reedyseth** [comment]
@Bobz-zg Do you have all up a running without issues ?

**Comment by Reedyseth** [comment]
> Second thing is a request if possible, and that's to update markup on management page so I can style them to my need, right now it's difficult to style it and it looks bad for me:

This commit closes this request: 2c5585d427b9a881106684cc11f5594c6a3f0140
